deploy/configmap.yaml

```diff
@@ -5,9 +5,10 @@ metadata:
 type: Opaque
 data:
   IMAGES: >
-    che-theia=eclipse/che-theia:0.3.19-nightly;
-    che-dev=eclipse/che-dev:nightly;
-    jwt-proxy=eclipse/che-jwtproxy:latest;
+    jwtproxy=eclipse/che-jwtproxy:latest;
```

Comment:
@amisevsk btw, che-jwtproxy is now on quay - https://quay.io/repository/eclipse/che-jwtproxy?tab=tags

Reply:
Ah, missed that development :(

Looks like cms on prod/preview need to be updated again.

Reply:
Actually, lets hold off on updating until we update the cm to reflect Che 7.2. The quay.io repo was created for 7.1 AFAICT.